<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPageForEmployer</class>
 <widget class="QWidget" name="RegisterPageForEmployer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
    
    border-radius: 25px;
	background-color: rgb(249, 249, 249);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="registrationLineEditCompanyNameEmployer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Golos Text</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #f9f9f9;
    font-size: 14px;
}</string>
            </property>
            <property name="placeholderText">
             <string>Название компании</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="registrationLineEditEmailEmployer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Golos Text</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #f9f9f9;
    font-size: 14px;
}</string>
            </property>
            <property name="placeholderText">
             <string>Электронная почта</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="registrationLineEditPasswordEmployer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Golos Text</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #f9f9f9;
    font-size: 14px;
}</string>
            </property>
            <property name="placeholderText">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="registrationPB_employer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>175</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Golos Text</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #0069d9;
}</string>
            </property>
            <property name="text">
             <string>Зарегистрироваться</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backToStatusPB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>175</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Golos Text</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    border-radius: 8px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #0069d9;
}</string>
            </property>
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
